### ISSUE TITLE
 'GeolocateControl' is not exported from '../utils/mapboxgl' (imported as 'MapboxGeolocateControl').

### ISSUE THREAD

**Issue by omguhh** [uber/react-map-gl#756]
First of, thank you so much for all the work you do on this library <3

I've updated to the latest version (4.1.0) and when running a production build, I keep running into this error with fullscreen control & geolocate control.

```
Creating an optimized production build...
Failed to compile.
./node_modules/react-map-gl/dist/esm/components/geolocate-control.js
Attempted import error: 'GeolocateControl' is not exported from '../utils/mapboxgl' (imported as 'MapboxGeolocateControl').
```

It's returning the same issue for FullscreenControl - any ideas? I could look into it in the evening. ^^

**Comment by xintongxia** [comment]
thanks for reporting, will fix soon.

**Comment by blair-anson** [comment]
I suspect this error is also related
```
Failed to compile.

./node_modules/react-map-gl/dist/esm/components/fullscreen-control.js
Attempted import error: 'FullscreenControl' is not exported from '../utils/mapboxgl' (imported as 'MapboxFullscreenControl').
```

**Comment by Pessimistress** [comment]
Fixed published in 4.1.1
